pkg/minikube/command/ssh_runner.go

```diff
@@ -406,7 +406,7 @@ func (s *SSHRunner) Copy(f assets.CopyableFile) error {
 		return nil
 	})
 
-	scp := fmt.Sprintf("sudo test -d %s && sudo scp -t %s", f.GetTargetDir(), f.GetTargetDir())
+	scp := fmt.Sprintf("sudo mkdir -p %s && sudo scp -t %s", f.GetTargetDir(), f.GetTargetDir())
```

Comment:
is there any code about "cp" that needs to be changed as well?

Reply:
not necessary, however, I was thinking about dropping `test -d` and we could call `scp` with the -r flag instead:
```
> man scp
...
-r      Recursively copy entire directories.  Note that scp follows symbolic links encountered in the tree traversal.`
```

Note, that if anybody expects it to fail, when the path does not exist, this will stop failing.